Cue/app/tabs/search/SearchListView.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+'use strict';
+
+import React from 'react'
+import { Navigator, View, Text, ListView, Image, Platform, Dimensions } from 'react-native'
+
+import { connect } from 'react-redux'
+
+import type { DeckMetaData, Card } from '../../api/types'
+
+import EmptyView from '../../common/EmptyView'
+import CueIcons from '../../common/CueIcons'
+
+import SearchListViewItem from './SearchListViewItem'
+
+const styles = {
+  empty: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  list: {
+  }
+}
+
+type Props = {
+  navigator: Navigator,
+  decks: Array<DeckMetaData>,
+}
+
+class SearchListView extends React.Component {
+  props: Props
+
+  ds: ListView.DataSource
+
+  state: {
+    dataSource: ListView.DataSource,
+    deviceOrientation: string

```

Comment:
Is `deviceOrientation` used here for anything? If not, you can remove it from the `state`.